public/kibi_timeline_vis_controller.js

```diff
@@ -83,15 +83,21 @@ define(function (require) {
               searchSource.source(_.compact([ group.labelField, group.startField, group.endField ]));
               searchSource.set('filter', queryFilter.getFilters());
 
+              // save label field name to `fielddata_fields`
+              // it is needed to display not indexed fields in case of multi-fields
+              searchSource._state.fielddata_fields = _.compact([ group.labelField ]);
+
               $scope.visOptions.groups.push({
                 id: group.id,
                 color: group.color,
                 label: group.groupLabel,
                 searchSource: searchSource,
                 params: {
                   //kibi params
-                  labelFieldSequence: fields[i].byName[group.labelField].path,
-                  startFieldSequence: fields[i].byName[group.startField].path,
+                  labelFieldSequence: fields[i].byName[group.labelField].path.length > 0
```

Comment:
When you want a value from an ordinary (indexed) field, you can get it in this way `fields[i].byName[group.labelField].path`. But it returns an empty array in case of multi-fields. A multi-field field name you can get from `[group.labelField]`.